pkg/util/xorm/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/grafana/grafana/pkg/util/xorm
 go 1.21.10
 
 require (
-	github.com/mattn/go-sqlite3 v1.14.19
```

Comment:
running `go mod tidy` updates this -- i assume it is ok/good to keep it updated!